dataactbroker/handlers/fileHandler.py

```diff
@@ -1113,6 +1113,18 @@ def restart_validation(submission):
 
         return JsonResponse.create(StatusCode.OK, {"message": "Success"})
 
+    def fail_validation(self, upload_id):
+        # update all validation jobs to "ready"
+        sess = GlobalDB.db().session
+
+        jobs = sess.query(Job).filter(Job.submission_id == upload_id).all()
```

Comment:
You can use `one_or_none()` here. If a user passes in an `upload_id` that doesn't correspond to any upload job in the database (since they can bypass the frontend by calling the API directly), an appropriate response and error code should be returned for that kind of error.